Misc/NEWS.d/next/Windows/2019-08-08-18-05-27.bpo-36266.x4eZU3.rst

```diff
@@ -0,0 +1 @@
+Add the module name in the formatted error message when DLL load fail happens during module import in :c:func:`_PyImport_FindSharedFuncptrWindows()`. Patch by Srinivas Nyayapati.
```

Comment:
`c:func` should only be used for the documented functions of the C API.

Reply:
I updated my message. Thank you.